cpu/native/include/netdev_tap.h

```diff
@@ -43,7 +44,8 @@ typedef struct netdev_tap {
     char tap_name[IFNAMSIZ];            /**< host dev file name */
     int tap_fd;                         /**< host file descriptor for the TAP */
     uint8_t addr[ETHERNET_ADDR_LEN];    /**< The MAC address of the TAP */
-    uint8_t promiscuous;                 /**< Flag for promiscuous mode */
+    uint8_t promiscuous;                /**< Flag for promiscuous mode */
+    uint8_t wired;                      /**< Flag for wired mode */
```

Comment:
Why not use `bool` here? I only recently learned that it is also 8 bit wide :-)

Reply:
Just consistency.